### ISSUE TITLE
Where is ./configure?

### ISSUE THREAD

**Issue by fantauzzi** [geany/geany-plugins#943]
I am following instructions in README.rst to build from source, after checking out tag 1.36.0. I would like to run `configure`, but cannot find it. I have installed autotools (and already successfully built Geany from source).

Where is the `./configure` script/command located?

**Comment by elextr** [comment]
The readme is for building from tarball, not from git, to build from git you have to run autogen.sh the first time to create configure

**Comment by fantauzzi** [comment]
After running `autogen.sh` I have been able to run `configure`, as you wrote.

I don't find a section "Installing from a Git clone" in the [readme](https://github.com/geany/geany-plugins).

**Comment by codebrainz** [comment]
> I don't find a section "Installing from a Git clone" in the readme. I assume you refer to the readme for Geany.

Correct, it was added in geany/geany#1087 for Geany's README, but no equivalent for Geany-Plugins'.

FWIW, if you want to use an actual release version, it's simpler to use [a release tarball](https://plugins.geany.org/downloads.html) which contains all the generated files that aren't checked into Git, and so also doesn't require additional Autotoools programs to generate those files. This is the type of build the README currently assumes.

Related: #379
